pkg/apis/psmdb/v1/psmdb_defaults_test.go

```diff
@@ -226,7 +226,7 @@ func TestSetSafeDefault(t *testing.T) {
 	cr := &api.PerconaServerMongoDB{
 		ObjectMeta: metav1.ObjectMeta{Name: "psmdb-mock", Namespace: "psmdb"},
 		Spec: api.PerconaServerMongoDBSpec{
-			CRVersion: version.Version,
+			CRVersion: "1.15.0",
```

Comment:
I think we should add or update this unit tests to cover these new unsafe flags for 1.16.0.